src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -94,77 +121,207 @@ static void OnFlyoutItemIsVisibleChanged(BindableObject bindable, object oldValu
+		/// </summary>
+		/// <param name="obj">The object that modifies if the navigation bar has a shadow.</param>
+		/// <returns>Returns if the navigation bar has a shadow.</returns>
 		public static bool GetNavBarHasShadow(BindableObject obj) => (bool)obj.GetValue(NavBarHasShadowProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetNavBarHasShadow']/Docs/*" />
+
+		/// <summary>
+		/// Controls whether the navigation bar has a shadow. 
+		/// By default the value of the property is <see langword="true"/> on Android, and <see langword="false"/> on other platforms.
+		/// </summary>
+		/// <param name="obj">The object that modifies if the navigation bar has a shadow.</param>
+		/// <param name="value">Manages if the navigation bar has a shadow.</param>
 		public static void SetNavBarHasShadow(BindableObject obj, bool value) => obj.SetValue(NavBarHasShadowProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetSearchHandler']/Docs/*" />
+		/// <summary>
+		/// Gets the integrated search functionality in Shell.
+		/// </summary>
+		/// <param name="obj">The object that modifies the Shell search functionality.</param>
+		/// <returns>The integrated search functionality in Shell.</returns>
 		public static SearchHandler GetSearchHandler(BindableObject obj) => (SearchHandler)obj.GetValue(SearchHandlerProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetSearchHandler']/Docs/*" />
+
+		/// <summary>
+		/// Sets the handler responsible for implementing the integrated search functionality in Shell.
+		/// Enabling it results in a search box being added at the top of the page.
+		/// </summary>
+		/// <param name="obj">The object that modifies the Shell search functionality.</param>
+		/// <param name="handler">Defines the integrated search functionality in Shell.</param>
 		public static void SetSearchHandler(BindableObject obj, SearchHandler handler) => obj.SetValue(SearchHandlerProperty, handler);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetTabBarIsVisible']/Docs/*" />
+		/// <summary>
+		/// Gets the tabs visibility in Shell.
+		/// </summary>
+		/// <param name="obj">The object that modifies the Shell tabs visibility.</param>
+		/// <returns>true if the tab bar is visible; otherwise, false.</returns>
 		public static bool GetTabBarIsVisible(BindableObject obj) => (bool)obj.GetValue(TabBarIsVisibleProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetTabBarIsVisible']/Docs/*" />
+
+		/// <summary>
+		/// Sets the tabs visibility in Shell.
+		/// The tab bar and tabs are visible in Shell applications by default. However, the tab bar can be hidden by setting the Shell.TabBarIsVisible attached property to false.
```

Comment:
This extra sentences can go into `remarks`.

Also, you could add some crefs here :P